BlockLooper/Source/BlockLooper.swift

```diff
@@ -26,9 +26,18 @@ import Foundation
 
 public typealias LoopableBlock = () -> Bool
 
-public class BlockLooper {
+open class BlockLooper {
+
+    struct Static {
+        static var instance: Helper?
+        static var token: Int = 0
+    }
+
+    private static var __once: () = {
+            Static.instance = Helper()
+        }()
     
-    public class func executeBlockWithRate(rate: NSTimeInterval, block: LoopableBlock) {
+    open class func executeBlockWithRate(_ rate: TimeInterval, block: @escaping LoopableBlock) {
```

Comment:
it would be better to update to Swift 3.0 syntax -- i.e: executeBlock(rate: TimeInterval, block: @escaping LoopableBlock)